tests/test_compressors/test_model_compressor.py

```diff
@@ -16,11 +16,25 @@
 
 import pytest
 from compressed_tensors.compressors.model_compressor import ModelCompressor
+from compressed_tensors.config.base import SparsityCompressionConfig
+from compressed_tensors.quantization.quant_config import QuantizationConfig
+from transformers import AutoModel, AutoConfig
+
+
+def ct_config_available():
```

Comment:
ct = compressed tensors? 
lets make the names verbose

Reply:
See: https://github.com/neuralmagic/compressed-tensors/pull/176